PocketKit/Sources/PocketKit/PocketNotificationService.swift

```diff
@@ -0,0 +1,170 @@
+        self.tracker = tracker
+        self.sessionManager = sessionManager
+        
+        // Init Braze with our information.
+        var configuration = Braze.Configuration(
+            apiKey: Keys.shared.brazeAPIKey,
+            endpoint: Keys.shared.brazeAPIEndpoint
+        )
+        
+        // Enable logging of general SDK information (e.g. user changes, etc.)
+        configuration.logger.level = .info
+        configuration.push.appGroup = "group.com.ideashower.ReadItLaterProAlp"
+        braze = Braze(configuration: configuration)
+        super.init()
+        
+        // Set up the in app message ui
+        let inAppMessageUI = BrazeInAppMessageUI()
+        inAppMessageUI.delegate = self
+        self.braze.inAppMessagePresenter = inAppMessageUI
+        
+        sessionManager.$currentSession.receive(on: DispatchQueue.main).sink { [weak self] session in
+            if let session = session {
+                // Braze SDK docs say this needs to be called from the main thread.
+                // https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/analytics/setting_user_ids/#assigning-a-user-id
+                self?.braze.changeUser(userId: session.userIdentifier)
+            }
+        }.store(in: &subscriptions)
+        
+        
+        let center = UNUserNotificationCenter.current()
+        // In the future we can setup other types of categories
+        // https://www.braze.com/docs/user_guide/message_building_by_channel/push/ios/notification_options/#summary-arguments
+        center.setNotificationCategories(Braze.Notifications.categories)
+        center.delegate = self
+        // We also ask for provisional authotization which is in iOS 12 and above and allows us to use push notifications that only post to the notification center
+        // https://www.braze.com/docs/user_guide/message_building_by_channel/push/ios/notification_options/#provisional-push
+        // Developer Note: Since we are registering provisional push, the user will never see the pop up to allow push notifications.
+        // This can make it challenging to test push notification registration flows because you will not see push registration appear, so you may want to remove this during active development.
+        center.requestAuthorization(options: [.badge, .sound, .alert, .provisional]) { granted, error in
+            print("Notification authorization, granted: \(granted), error: \(String(describing: error))")
+        }
+    }
+    
+    /**
+     Proxy function to braze to register a device token
+     */
+    public func register(deviceToken: Data) {
+        //TODO: Also register the token with the Pocket Instant Sync endpoint
+        braze.notifications.register(deviceToken: deviceToken)
+    }
+    
+    /**
+     Proxy function to braze to handle background notifications
+     */
+    public func handleBackgroundNotifcation(didReceiveRemoteNotification userInfo: [AnyHashable : Any],
```

Comment:
Let's adopt the style where we have a newline after the opening paren for these params (see below)